services/wallet/router/fees.go

```diff
@@ -24,10 +24,25 @@ const (
 	GasFeeHigh
 )
 
+type MaxFees struct {
+	Low    *big.Int `json:"low"`
+	Medium *big.Int `json:"medium"`
+	High   *big.Int `json:"high"`
+}
+
+type SuggestedFees struct {
+	GasPrice             *big.Int   `json:"gasPrice"`
+	BaseFee              *big.Int   `json:"baseFee"`
+	MaxFees              *MaxFees   `json:"maxFees"`
+	MaxPriorityFeePerGas *big.Int   `json:"maxPriorityFeePerGas"`
+	L1GasFee             *big.Float `json:"l1GasFee,omitempty"`
+	EIP1559Enabled       bool       `json:"eip1559Enabled"`
+}
+
 // //////////////////////////////////////////////////////////////////////////////
-// TODO: remove `SuggestedFees` struct once new router is in place
+// TODO: remove `SuggestedFeesGwei` struct once new router is in place
 // //////////////////////////////////////////////////////////////////////////////
-type SuggestedFees struct {
+type SuggestedFeesGwei struct {
```

Comment:
Why do we need two types, can we go with `SuggestedFeesGwei` only?

Reply:
I didn't want to change old codes "yet"